tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -12,35 +13,46 @@
 from .onnx_model_utils import get_optimization_level
 
 
+class OptimizationStyle(enum.Enum):
+    Default = 0
```

Comment:
open to better names than `Default`

Reply:
Maybe 'Fixed'? 'Default' doesn't have much meaning in this context.